app/views/smart_answers/_debug.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if @presenter.debug? %>
```

Comment:
This is currently failing because `debug?` isn't a method on `SmartDownPresenter`.